test/google_scraping/accounts_test.exs

```diff
@@ -0,0 +1,163 @@
+defmodule GoogleScraping.AccountsTest do
+  use GoogleScraping.DataCase
+
+  alias GoogleScraping.Accounts
+  alias GoogleScraping.Accounts.Schemas.{User, UserToken}
+
+  describe "get_user_by_email/1" do
+    test "when the email does not exist, does not return the user" do
+      assert Accounts.get_user_by_email("unknown@example.com") == nil
+    end
+
+    test "when the email exists, returns the user" do
+      %{id: id} = user = insert(:user)
+      assert %User{id: ^id} = Accounts.get_user_by_email(user.email)
+    end
+  end
+
+  describe "get_user_by_email_and_password/2" do
+    test "when the email does not exist, does not return the user" do
+      assert Accounts.get_user_by_email_and_password("unknown@example.com", "hello world!") == nil
+    end
+
+    test "when password is invalid, does not return the user" do
+      user = insert(:user)
+      assert Accounts.get_user_by_email_and_password(user.email, "invalid") == nil
+    end
+
+    test "when email and password are valid, returns the user" do
```

Comment:
While testing, we always write the happy case first, then write invalid/unhappy cases. So it case should move first. Same others tests